### ISSUE TITLE
Ignore on same line failure

### ISSUE THREAD

**Issue by ghost** [foundeo/fixinator#6]
In the README.md in the [Ignoring issues in code](https://github.com/foundeo/fixinator#ignoring-issues-in-code) section it states you can add ignores on the line or above but it only seems to work on above the line.

> The comment must be on the same line as the issue, or on the line above the issue.

It does not ignore if it is on the same line.

Example
```
var password = "test2021!"; //ignore:plain-text-key
```
Response:
```
[MEDIUM] plain-text-key found on line 7 of JunkTest.cfc high confidence
/JunkTest.cfc:7
    7: var password = "**********"; //ignore:plain-text-key
```

File *JunkTest.cfc*
```
component {
	function run()
	{
		describe("Test",
			function()
			{
				var password = "test2021!"; //ignore:plain-text-key
			});
	}
}
```

**Comment by pfreitag** [comment]
Thanks @rosscupidmedia for providing the example cfc, confirmed I am seeing the same issue.

**Comment by pfreitag** [comment]
Fixed in version 2.0.3
